compiler/src/dotty/tools/backend/jvm/GenBCode.scala

```diff
@@ -242,16 +242,16 @@ class GenBCodePipeline(val entryPoints: List[Symbol], val int: DottyBackendInter
     } // end of class BCodePhase.Worker1
 
     /*
-     *  Pipeline that takes ClassNodes from queue-2. The unit of work depends on the optimization level:
+     *  Pipeline that takes ClassNodes from queue-2. The unit of work depends on the optimisation level:
      *
-     *    (a) no optimization involves:
+     *    (a) no optimisation involves:
      *          - converting the plain ClassNode to byte array and placing it on queue-3
      */
     class Worker2 {
       // lazy val localOpt = new LocalOpt(new Settings())
 
-      def localOptimizations(classNode: ClassNode): Unit = {
-        // BackendStats.timed(BackendStats.methodOptTimer)(localOpt.methodOptimizations(classNode))
+      def localOptimisations(classNode: ClassNode): Unit = {
+        // BackendStats.timed(BackendStats.methodOptTimer)(localOpt.methodOptimisations(classNode))
```

Comment:
I propose not to rename the method here. This is how method is actually called in the backend.